dashboard/src/components/Config/AppRepoList/AppRepoList.tsx

```diff
@@ -208,7 +202,7 @@ function AppRepoList() {
               <LoadingWrapper
                 className="margin-t-xxl"
                 loadingText="Fetching Application Repositories..."
-                loaded={!isFetching}
+                loaded={!isFetchingElem.repositories}
```

Comment:
Not clear to me why this change is here? Looks like we're making the condition for loaded be more specific, to only depend on the repositories?

Reply:
Are we using this `isFetchingElem` elsewhere? I don't see any `isFetchingElem.secrets` for example in the code, for example.
Just out of curiosity, I'm keen to have more granularity here.

Reply:
> Looks like we're making the condition for loaded be more specific, to only depend on the repositories?

Reply:
Yes, this is because before we were fetching secrets and repos here (now just repos). Without this change, the apprepo form modal is closed just after opening it. This is because we are requesting secrets at that moment and the parent component, this loadingWrapper, was detecting the change and rendering the loading gif instead.